packages/codemods/src/maybe.ts

```diff
@@ -21,27 +21,24 @@ const _makeMaybe = <T>(): MB<T> => {
 };
 
 export interface Just<T> extends MB<T> {
-  value: T;
+  value: NonNullable<T>;
   just: true;
 }
 
 export interface Nothing<T> extends MB<T> {
   just: false;
 }
 
-export type Maybe<T> = Just<NonNullable<T>> | Nothing<NonNullable<T>>;
+export type Maybe<T> = Just<T> | Nothing<T>;
 
 // Need to use assign so that the object remains correctly bound to the methods.
 export const Nothing = <T>(): Nothing<T> => Object.assign<MB<T>, { just: false }>(_makeMaybe(), { just: false });
 
-export const Just = <T>(value: NonNullable<T>): Just<NonNullable<T>> => {
+export const Just = <T>(value: NonNullable<T>): Just<T> => {
```

Comment:
I wonder if it would be better to call this `Only` since we already have an unrelated thing called `just`? (missed this in the PR where it was added.) Though I guess you could argue that the confusion risk is not super high since the context is different.

Reply:
The just property is different than the Just function. I like having Just as both a function and a type. I'd be happy to rename it to something more explicit, but I'll do that in another PR.

Reply:
Also the just property is a way of ensuring that the types, Just and Nothing are distinguishable. So that property name would end up changing too. 

It's following the way that Maybe works in some other libraries, but another naming convention would be to rename it Some which follows Rust's option type.